service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseAccessService.java

```diff
@@ -33,6 +38,9 @@ public class CaseAccessService {
     private final SystemUserService systemUserService;
     private final OrganisationService organisationService;
 
+    private final List<String> excludedCaseRoles = List.of(ALLOCATED_JUDGE.getRoleName(), HEARING_JUDGE.getRoleName(),
```

Comment:
Could we name this something more descriptive for the long term?

E.g. excludedInternalWACaseRoles

That way we could remove the comment on the filter as that might get out of date in future and be miss leading